.travis.yml

```diff
@@ -25,27 +25,27 @@ matrix:
     - python: '3.7'
       env:
         - TOXENV="py37-linux"
-        - BOTO_CONFIG="/dev/null"
+        - BOTO_CONFIG="/dev/null"  # XXX: why is this here?
       dist: xenial
-      sudo: true
+      sudo: true  # XXX: why is this here vs. others?
```

Comment:
@menshikh-iv can you comment here as well?

Reply:
same as https://github.com/RaRe-Technologies/gensim/pull/2814#discussion_r417767513